### PR TITLE
started development page

### PR DESCRIPTION
**Pull request by lucadelu**
This is only the starting pull request to speak about this page, don't merge it

Do you like the structure?

### CODE REVIEW

**Comment by nbozon** [comment]
Sounds good to me !
I will  also tweak the corresponding template to show useful links for developers on the right column

**Comment by mlennert** [comment]
Looks good to me too. Maybe core development could be a bit higher up in the order of sections, but I do understand that it probably concerns the fewest people. Just seems a bit odd to have it at the very end.

**Comment by lucadelu** [comment]
> Sounds good to me !
> I will also tweak the corresponding template to show useful links for developers on the right column

Ok, I can tweak the template too

File: content/contribute/development.md [hunk 1]
```diff
@@ -1,9 +1,86 @@
...
 
+The strength and success of GRASS GIS relies on the **user community** to improve and extend the GRASS GIS capabilities.
+GRASS GIS is developed and supported by a [worldwide developers team](https://github.com/OSGeo/grass/blob/master/contributors.csv)
 
+Many people have contributed to the GRASS GIS. Without them, GRASS GIS would not exist in its current form.
+The authors of the individual programs are listed at the end of the respective manual pages.
+
+## Get Involved
+
+### Report bug
```
Comment (neteler) [inline content/contribute/development.md]:
```suggestion
### Report a bug or enhancement wish
```

File: content/contribute/development.md [hunk 2]
```diff
@@ -1,9 +1,86 @@
...
+<img src="../../images/bugreport.svg" width="30%" alt="" style="float:right">
+Bug reports and wishes can be submitted in the [GRASS GIS bugtracking system](http://trac.osgeo.org/grass) (GRASS-trac).
```
Comment (neteler) [inline content/contribute/development.md]:
```suggestion
Bug reports and wishes can be submitted in the [GRASS GIS bugtracking system](https://github.com/OSGeo/grass/issues).
```

File: content/contribute/development.md [hunk 3]
```diff
@@ -1,9 +1,86 @@
...
+
+All trackers' new submissions will be automatically forwarded to the GRASS GIS development mailing list.
```
Comment (neteler) [inline content/contribute/development.md]:
```suggestion
All new submissions in the tracker will be automatically forwarded to the GRASS GIS development mailing list.
```

File: content/contribute/development.md [hunk 4]
```diff
@@ -1,9 +1,86 @@
...
+
+###### Some recommendation
```
Comment (neteler) [inline content/contribute/development.md]:
```suggestion
###### Some recommendations
```

File: content/contribute/development.md [hunk 5]
```diff
@@ -1,9 +1,86 @@
...
+<small>
+<ul>
+ <li>Check if the bug is still present in the current stable release version or "master" before reporting. If you use an older version, please consider [upgrading](/download)</li>
```
Comment (neteler) [inline content/contribute/development.md]:
```suggestion
 <li>Check if the bug you found is still present in the current stable release version or, better, in the "master" version before reporting. If you use an older version, please consider to [upgrade](/download) first</li>
```

File: content/contribute/development.md [hunk 6]
```diff
@@ -1,9 +1,86 @@
...
+ <li>Make sure that the developers have all the information needed to recreate the bug (e.g. explain how to reproduce the bug using the [North Carolina example dataset]())</li>
+ <li>Report only a single bug with each request</li>
+ <li>Before reporting a bug, please check that it is yet unknown: GRASS-trac bug tracking system [search page](http://trac.osgeo.org/grass/report)</li>
```
Comment (neteler) [inline content/contribute/development.md]:
```suggestion
 <li>Before reporting a bug, please check that it hasn't already been reported: [GRASS GIS bugtracking system](https://github.com/OSGeo/grass/issues)</li>
```

File: content/contribute/development.md [hunk 7]
```diff
@@ -1,9 +1,86 @@
...
+Further details about how to gain access to our [GitHub Addons repository](https://github.com/OSGeo/grass-addons/) you can find in [this document](https://trac.osgeo.org/grass/wiki/HowToContribute#WriteaccesstotheGRASS-Addons-SVNrepository).
+
+### Contribute to the core
+<img src="../../images/core_devel.svg" width="30%" alt="" style="float:right;padding-left:10px">
+The core consists in the libraries used by the GRASS GIS modules and GUI and the code used to develop the User Graphic Interface and the most important modules
+
+It may be a good idea to send a small posting to the ​[GRASS developers mailing list](http://lists.osgeo.org/mailman/listinfo/grass-dev) to announce your activities
+(maybe someone will join you!). Please read [how to write source code](https://trac.osgeo.org/grass/wiki/HowToProgram). 
+
+We appreciate if you can first [fix ​bugs](http://trac.osgeo.org/grass/query?status=%21closed&type=defect&order=id&desc=1) etc.
```
Comment (neteler) [inline content/contribute/development.md]:
```suggestion
We'd appreciate if you could support us with [fixing of ​bugs](https://github.com/OSGeo/grass/issues) etc.
```

File: themes/grass/layouts/contribute/devel.html [hunk 8]
```diff
@@ -0,0 +1,44 @@
...
+      <div class="row">
+        <div class="col-lg-4">
+          <div class="p-4 bg-lgr sticky-top">
+            <nav class="sidebar-menu">
+              <div class="list-styled" style="margin-left: 2em">
+                <h3>Useful links</h3>
+              </div>
+            </nav>
+            <ul>
+              <li><a href="http://trac.osgeo.org/grass" target="_blank">Bug Tracker</a></li>
```
Comment (neteler) [inline themes/grass/layouts/contribute/devel.html]:
```suggestion
              <li><a href="https://github.com/OSGeo/grass/issues" target="_blank">Bug Tracker</a></li>
```

**Comment by veroandreo** [comment]
@lucadelu maybe you can merge this one, so we see how it displays and then keep working on it

**Pull request merged**